pkg/datarepair/checker/checker.go

```diff
@@ -30,31 +30,42 @@ var (
 
 // Config contains configurable values for checker
 type Config struct {
-	Interval time.Duration `help:"how frequently checker should audit segments" default:"30s"`
+	Interval      time.Duration `help:"how frequently checker should audit segments" releaseDefault:"30s" devDefault:"0h0m10s"`
+	IrrdbInterval time.Duration `help:"how frequently irrepairable checker should audit segments" releaseDefault:"15s" devDefault:"0h0m5s"`
```

Comment:
It shouldn't have db in name, it doesn't have to be related to any DB. For now maybe just use IrreparableInterval.

Also don't use `audit` in the description because it can be confused with a real audit. Maybe something like `how frequently irrepairable should recheck lost pieces` but maybe you will have better idea.

Reply:
done